.travis.yml

```diff
@@ -3,7 +3,5 @@ sudo: required
 language: java
 jdk: oraclejdk8
 
-before_install:
-  - sudo pip install codecov
 after_success:
-  - codecov
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
I switched to the bash uploader for a little faster processing during CI. But both options work